pkgs/stdenv/generic/check-meta.nix

```diff
@@ -390,6 +390,48 @@ let
     # -----
     else { valid = "yes"; });
 
+
+  commonMeta = { validity, attrs, pos ? null, references ? [ ] }:
```

Comment:
Please add some documentation comment about the function

Reply:
added something